test/valid-file-path.test.js

```diff
@@ -5,7 +5,8 @@ describe('Linux - Correct', () => {
     'a/my_images/IMG--__23~123.min.jpeg',
     'modules/admin/foo_bar/test--__23~123.html.liquid',
     'modules/admin cms/foo bar/test (Screenshot)%20@.liquid',
-    'a/foo/test+99.zip'
+    'a/foo/test+99.zip',
+    "modules/homepage/public/assets/images/blog/second-half-2016/Beginner's-Running-Shoes.jpg"
   ];
```

Comment:
'modules\\homepage\\public\\assets\\images\\blog\\2017\\Training for Long Distance Swimming & Triathlons.jpg' -> linux version